.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Before you get started
+- [ ] 🙋‍♀️ Create an issue to discuss what you are doing to do!
+
+---
+
+## Description
+- Closes #issue_number
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 
+
+## Type of change
+Please delete options that are not relevant.
+
```

Comment:
Is the list "complete"? We defined a couple of more issue labels, e.g. type:testing. On the other side, we should restrict to the most relevant "main category" types and avoid having each module defined as a type here.

Reply:
For a pull request, I think this list is complete. Issues can have a more diverse set of labels, but you wouldn't do a pull request using them. In the end, the pull request can also receive those labels if necessary.

Reply:
I agree!